### DISCUSSION TITLE
MWAA shell scripts

### DISCUSSION THREAD

**Discussion by match-gabeflores** [tomasfarias/airflow-dbt-python#116] [General]
fyi - Amazon recently announced using shell scripts in MWAA. so you can install the dbt cli (i think) and call it directly from bash operator. 
https://docs.aws.amazon.com/mwaa/latest/userguide/using-startup-script.html

**Comment by tomasfarias** [comment]
Thanks for sharing.

Probably worth updating the docs and making a mention; accepting PRs for this. However, airflow-dbt-python's value proposition has gone beyond allowing you to run dbt in MWAA, so I'm not really taking any action items from this.
